CMakeLists.txt

```diff
@@ -1,11 +1,16 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Is this required? I'm tentative to bump the minimum required here without a really good reason.